server/game/cards/10-SoD/AtPrinceDoransBehest.js

```diff
@@ -24,10 +24,10 @@ class AtPrinceDoransBehest extends PlotCard {
         });
     }
 
-    getPlotSourceForPhase(card) {
-        return (this.game.currentPhase !== 'plot') ? 
-            ['plot deck', 'scheme plots'].includes(card.location) :
-            card.location === 'plot deck';
+    checkPlotForPhase(card) {
+        return (this.game.currentPhase !== 'plot') ?
+            card.location === 'plot deck' :
+            card.location === 'plot deck' && !card.hasTrait('scheme');
```

Comment:
This will prevent Behest from revealing a Scheme during the plot phase, which we'd only want to prevent if the player is using Rains as their agenda. Checking also that they can select schemes should solve it.

Reply:
I think the solution to the below comment will fix this by default